dev_environment/README.md

```diff
@@ -28,35 +31,69 @@ Here is a visual overview:
   on which eBPF programs are running and how they are configured).
-* To see the eBPF program metrics, browse to `http://localhost:33000` on the
+11. To see the eBPF program metrics, browse to `http://localhost:33000` on the
   host and login to Grafana with the default username and password of `admin`.
   After logging in you will be able to view the preconfigured dashboards.
+
+# Trying out L3AF on an Azure VM (Ubuntu 20.04.4 LTS)
+The following host prerequisites and installation instructions are for trying out L3AF on an Azure VM running an Ubuntu 20.04.4 LTS server.
+
+## Host Prerequisites
+
+* [L3AFD source code](https://github.com/l3af-project/l3afd)
+* [hey](https://github.com/rakyll/hey) or any HTTP load generator
+* A web browser
+* [Go for Linux](https://go.dev/doc/install) version 1.18.4 or greater
+
+## Installation Instructions
+1. Clone the [l3afd](https://github.com/l3af-project/l3afd.git) and [l3af-arch](https://github.com/l3af-project/l3af-arch.git)
+Github repositories to your Azure VM.
+2. Download and install Go for Linux version 1.18.4 or greater at https://go.dev/doc/install. Make sure that you do **not** use `sudo apt install`,
+because this command will install version 1.13 but the installation requires version 1.17 at the minimum.
+3. Edit `config.yaml` to point to the [L3AFD repository](https://github.com/l3af-project/l3afd) on your host machine.
+4. Run `install.sh` to install package dependencies, Grafana, Prometheus, the eBPF source code you need to build your eBPF programs against,
+and the eBPF package repository.
+5. On the Azure VM, go to your L3AFD directory and run `go install .`
+6. On the Azure VM, go to `~/go/bin` and run `l3afd` as root:
+  `sudo ./l3afd --config /vagrant/cfg/l3afd.cfg`.
+
+## Trying out your L3AFD Server
+1. Start test web servers by running the following commands on your Azure VM: `go run l3af-arch/dev_environment/code/web-server.go -port 8080 > /var/log/web-server1.log 2>&1 &` and `go run l3af-arch/dev_environment/code/web-server.go -port 8081 > /var/log/web-server1.log 2>&1 &`.
+2. Verify that no eBPF programs are running by querying the
+L3AFD server from your laptop: `curl http://<ip-address-of-your-azure-vm>:8899/kfs/eth0`. This
+command assumes `eth0` is a valid network interface on the VM and should return with an empty set.
+3. Verify that you can send traffic to one of the test web servers running on the Azure VM with this command:
+`hey -n 200 -c 20 http://<ip-address-of-your-azure-vm>:8080`. This command should return a latency distribution histogram that shows
+ most traffic clustered near the top of the graph at very low latency.<p align="center"><img src="https://user-images.githubusercontent.com/106849610/179866166-597bef0d-2f5f-4ae7-89ee-1acdda5fd060.png" width="400" height="200"/></p>
+4. Load and run the ratelimiter eBPF program in the kernel with the following command: `curl -X POST http://<ip-address-of-your-azure-vm>:7080/l3af/configs/v1/update -d "@payload.json"`.
```

Comment:
The path to `payload.json` needs to be specified. There are multiple eBPF programs contained in this file (ratelimiter, connection_limit, and flow exporter). It may be necessary to rephrase it or create a new json file containing only the ratelimiter program. The values for the keys `host_name` and `iface` should be updated.

Reply:
I think we should supply a couple of different payload.json options that somebody can select. 

1. Can be simple with single program such as rate limiter
2. Can be multiple programs chained together